src/podman.scss

```diff
@@ -86,4 +86,39 @@
     .show-only-when-narrow {
         display: none;
     }
+
+    .ct-table {
+        td {
+            &[data-label=Owner], &[data-label=CPU], &[data-label=Memory], &[data-label="Disk space"], &[data-label="Used by"] {
+                white-space: nowrap;
+            }
+
+            &[data-label=Owner], &[data-label=CPU], &[data-label=Memory], &[data-label="Disk space"], &[data-label="Created"] {
+                text-align: right;
+            }
+
+            &[data-label=CPU] {
+                min-width: 6ch;
+            }
+
+            &[data-label=Memory] {
+                min-width: 14ch;
```

Comment:
This would not work when localized. Data labels are localized as well. But also, cannot we just pass this as property through JS. We define these columns so it should be possible to tell them, if they are for example left or right aligned.

Reply:
Yeah, I was afraid of that.

It's more than being right or left aligned, of course. But for whatever we can't represent with PF, we could add classes.

Reply:
I mentioned it last night when I was hastily pushing up this draft PR (as it was late): "but this could be done better, probably inside the table itself".

I may need help with this to locate where that should be done. I ran out of time last night to do this properly, so sending up a draft to demonstrate how it should be is better than keeping it to myself.